lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -134,12 +141,60 @@ var WebMapServiceCatalogItem = function(terria) {
      */
     this.displayDuration = undefined;
 
+    /**
+     * Gets or sets a value indicating whether the user's ability to change the display properties of this
+     * catalog item is disabled.  For example, if true, {@link WebMapServiceCatalogItem#styles} should not be
+     * changeable through the user interface.
+     * This property is observable.
+     * @type {Boolean}
+     * @default false
+     */
+    this.disableUserChanges = false;
```

Comment:
Interesting. There's a legacy `this.disableUserChanges` on `TableCatalogItem` too, which I see now is completely unused. I'll submit a PR to get rid of that too.